### DISCUSSION TITLE
NMI watchdog: BUG: soft lockup - CPU#1 stuck for 54s! [rsync:29733]

### DISCUSSION THREAD

**Discussion by zhuxi1965** [Security-Onion-Solutions/securityonion#10467] [Unsupported Versions]
I have download  SecurityOnion 2.3.250-20230519 ISO image,Install to KVM.
When the process to post-installation scripts,got the error "NMI watchdog: BUG: soft lockup - CPU#1 stuck for 54s! [rsync:29733]".
I have try some times,but got same error.
My kvm configuratio is:CPU 4 Core,RAM 20GB,Disk 300GB.
<img width="379" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/29396844/b6f8c49c-e74e-473e-8c06-2dfab767fb40">

**Comment by dougburks** [comment]
Is it fully stuck at that point or does it continue installation?

Did you verify the ISO image as shown at https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md?

Have you tried a standard CentOS 7 ISO image to see if that behaves differently?
